lib/warren/railtie.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Warren
+  # Railtie for automatic configuration in Rails apps. Reduces the need to
+  # modify the application when adding Warren.
+  # @see https://api.rubyonrails.org/classes/Rails/Railtie.html
+  class Railtie < Rails::Railtie
+    # initializer "railtie.configure_rails_initialization" do
+
+    # end
```

Comment:
We can remove the commented lines, if they are not needed anymore.

Reply:
Good point.